janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -942,7 +942,11 @@ public boolean apply(@Nullable String s) {
     public static final Duration TRANSACTION_LOG_DEFAULT_TTL = Duration.ofDays(7);
 
     public static final ConfigOption<String> LOG_BACKEND = new ConfigOption<>(LOG_NS,"backend",
-            "Define the log backed to use",
+            "Define the log backed to use. A reserved shortcut `default` can be used to use graph's storage backend to manage logs. " +
+                "Custom log implementation can be specified by providing " +
```

Comment:
**🥜 nitpick:** ‏I would say "A custom log implementation"

Reply:
Thank you @cdegroc ! Applied your comments into the commit